grafana/provisioning/datasources/postgres.yml

```diff
@@ -0,0 +1,17 @@
+apiVersion: 1
+
+datasources:
+  - name: Postgres
+    type: postgres
+    url: postgres
```

Comment:
The URL to our postgres is just `postgres` thanks to the address we created [here](https://github.com/okfn-brasil/serenata-de-amor/pull/484/files#diff-4e5e90c6228fd48698d074241c2ba760R49)